Source: Ingmar Vanhassel
Upstream: Not yet, will submit.
Reason: We install libruby as libruby18.so, FindRUBY.cmake only looks for libruby1.8.so (which is how Debian installs it)

From 8247b143ac05af55a43335586a6b6e9b1a04a227 Mon Sep 17 00:00:00 2001
From: Ingmar Vanhassel <ingmar@exherbo.org>
Date: Wed, 14 Jan 2009 10:11:57 +0100
Subject: [PATCH] Look for libruby18 etc as well, which is how it's installed on Exherbo, Gentoo & FreeBSD.

---
 cmake/modules/FindRUBY.cmake |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/cmake/modules/FindRUBY.cmake b/cmake/modules/FindRUBY.cmake
index b5d3c0f..2696c1b 100644
--- a/cmake/modules/FindRUBY.cmake
+++ b/cmake/modules/FindRUBY.cmake
@@ -61,7 +61,7 @@ IF(RUBY_EXECUTABLE)
   	)
 
 	FIND_LIBRARY(RUBY_LIBRARY
-		NAMES ruby${RUBY_VERSION} ruby msvcrt-ruby${RUBY_VERSION_NODOT}
+		NAMES ruby${RUBY_VERSION} ruby${RUBY_VERSION_NODOT} ruby msvcrt-ruby${RUBY_VERSION_NODOT}
 		PATHS ${RUBY_POSSIBLE_LIB_PATH} ${RUBY_RUBY_LIB_PATH}
   	)
 
-- 
1.6.1

